pymeasure/instruments/srs/sr830.py

```diff
@@ -553,14 +561,63 @@ def buffer_measure(self, count, stopRequest=None, delay=1e-3):
         ch2[index:count] = self.get_buffer(2, index, count)
         return (ch1.mean(), ch1.std(), ch2.mean(), ch2.std())
 
+    def buffer_measure_fast(self, buffer_size, fast=True, timeout=60,):
+        """Measure the buffer in either the fast or slow
+        data transfer mode. Returns X, Y values in the fast
+        mode as np.arrays.
```

Comment:
You can write about the return values after the parameters, with `:return: X, Y values as tuple of numpy arrays`.